requirements/edx/base.txt

```diff
@@ -218,5 +218,5 @@ edx-sga==0.6.2
 # Redis version
 redis==2.10.6
 
-# Completion
-edx-completion==0.1.10
+# Completion - using custom.txt for Github branch
+# edx-completion==0.1.10
```

Comment:
Where did 0.1.11 - 0.1.14 go?  Did they just never get used?  Did you look into the changes they contain, and whether they are safe?

Reply:
I decide to bump it after checking the commit log, looks fine to me:
```
4c96b3f (HEAD -> paulo/oauth, tag: 0.1.15, viadanna/paulo/oauth, opencraft/paulo/oauth) v0.1.15 add Oauth2 auth
7f8ee3d (tag: 0.1.14, viadanna/master, master) Merge pull request #33 from edx/cliff/fix-version-0.1.14
d2dab58 (origin/cliff/fix-version-0.1.14) Version bump to 0.1.14
43273f1 Merge pull request #32 from edx/cliff/allow-submission-for-any-user
9ae2a15 (origin/cliff/allow-submission-for-any-user) Fix quality
90df5ad Allow submitting by user_ids as well as users.
5a62a83 Allow xblocks to submit completions for arbitrary users.
7120420 (tag: 0.1.13) Merge pull request #31 from edx/jeskew/fix_v1_namespace
fe700c3 Bump version to 0.1.13
38150ef Add missing v1 app_name to fix url resolution.
d6bd593 (tag: 0.1.12) Merge pull request #29 from edx/ahsan/LEARNER-6194-add-jwt-authentication-classes
0cbb4f0 Added JWT authentication classes We need to add JWT authentication classes for mobile authentication of end points LEARNER-6194
d3781b9 Merge pull request #30 from edx/jeskew/fix_django20_deprecation_warning
616a9f2 (tag: 0.1.11) Bump version to 0.1.11
5cffa10 Fix Django20 deprecation warning.
```